src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -752,16 +752,6 @@ public function provideAbbreviationsForHiddenCommands()
         ];
     }
 
-    public function testFindAmbiguousCommandsIfAllAlternativesAreHidden()
```

Comment:
the test is already present before named `testFindWithAmbiguousAbbreviationsFindsCommandIfAlternativesAreHidden()`